src/app/Home.module.css

```diff
@@ -7,7 +7,7 @@
     margin-right: 1em;
 }
   
-h2 {
+.landingBackground h2 {
     /* TEMPORARY color till global theme is placed*/
     color: #057cde;

```

Comment:
[Please Change]: The solution for this issue is to remove these temporary colors; they overwrite the default theme and should not have been added.

Reply:
Done